### ISSUE TITLE
Orders not update in Production mode

### ISSUE THREAD

**Issue by jcantalogo** [peatio/peatio#513]
Please, I read n time application.yml and it is correct. All daemons are running perfectly but pusher is not connected and trade screen is not updating. 
Anybody could help me !??? please.

**Comment by valuead** [comment]
We can help you with full setup at 3000 US

**Comment by hpyhacking** [comment]
please confirm reds, rabbitmq and all peatio daemons are started

**Comment by karthickms** [comment]
Yes. I had the same issue. what the steps to finish it. Please hand to me
